docs/old-reference-guide/modules/axon-framework-commands/pages/modeling/aggregate.adoc

```diff
@@ -8,6 +8,31 @@ ensuring changes to one part don’t mess up others.
 Aggregates are commonly used to model the core business entities in a system based on link:https://www.axoniq.io/concepts/domain-driven-design[Domain-Driven Design] and link:https://www.axoniq.io/concepts/cqrs-and-event-sourcing[CQRS].
 This chapter will cover the basics on how to implement one in Axon Framework.
 
+== Aggregate identifiers
+
+Each aggregate has an identifier that uniquely identifies it within the context.
+Events published from this aggregate will contain this identifier,
+so that Axon Framework can later load the aggregate from its event stream.
+
+Commands that target aggregates contain a reference to this identifier.
+Axon Framework will load the events for the aggregate with this identifier,
+replay them on an empty instance, and invoke the command.
```

Comment:
Perhaps add one sentence to conclude that this behavior of stream retrieval, empty instance creations, and replaying of the events, is event sourcing in action?
```suggestion
replay them on an empty instance, and invoke the command. In doing so, this style supports the event sourcing of aggregates.
```

Reply:
Agreed, but I modified it a little bit :)